src/Tooltip.js

```diff
@@ -154,14 +180,16 @@ class Tooltip extends React.Component {
 
     return (
       <TetherContent
-        onMouseOver={this.onMouseOverTooltip}
```

Comment:
extra props are not part of the clone children stuff that goes on in TetherContent. I think it makes more sense to apply these event handlers to the children directly.